params/types/coregeth/chain_config.go

```diff
@@ -163,7 +163,7 @@ type CoreGethChainConfig struct {
 	EIP1706FBlock *big.Int `json:"eip1706FBlock,omitempty"`
 
 	// https://github.com/ethereum/EIPs/pull/2537: BLS12-381 curve operations
-	EIP2537FBlock *big.Int `json:"eip2315FBlock,omitempty"`
+	EIP2537FBlock *big.Int `json:"eip2537FBlock,omitempty"`
```

Comment:
Note that this is fixing a typo of mine.  Although an error like this could have been a major PIA if it were for a "production" chain, or if the mixed-up values were on different chains or forks, none of this is the case; the features are active only on YoloV2 and occur at the same block (and always together, AFAIK).